services/gateway-api/src/circuit_breaker/meta_breaker.py

```
@@ -0,0 +1,357 @@
+                logger.warning("Meta access token not configured")
+                return False
+
+            # Check if API is reachable
+            url = "https://graph.facebook.com/v18.0/me"
+            params = {"access_token": breaker.access_token}
+
+            async with aiohttp.ClientSession() as session:
+                async with session.get(url, params=params, timeout=aiohttp.ClientTimeout(total=5)) as response:
+                    return response.status == 200
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)